examples/ecs/ecs_guide.rs

```diff
@@ -211,9 +211,13 @@ fn exclusive_player_system(world: &mut World) {
     };
     // Randomly add a new player
     if should_add_player {
+        println!(
+            "An exclusive player {} has joined the game!",
```

Comment:
```suggestion
            "Player {} has joined the game!",
```

The term “exclusive” in the function name refers to itself being an exclusive system. It doesn't have to do with the player.

Reply:
Good point, I've changed it